packages/mobile/src/networkInfo/saga.ts

```diff
@@ -37,11 +37,15 @@ const isErrorResponse = (response: UserLocationData | ErrorResponse): response i
 function* subscribeToNetworkStatus() {
   const networkStatusChannel = yield createNetworkStatusChannel()
   let connectionInfo: NetInfoState = yield call(NetInfo.fetch)
-  yield put(setNetworkConnectivity(isConnected(connectionInfo)))
+  let isNetworkConnected = isConnected(connectionInfo)
+  yield put(setNetworkConnectivity(isNetworkConnected))
+  Logger.setIsNetworkConnected(isNetworkConnected)
   while (true) {
     try {
       connectionInfo = yield take(networkStatusChannel)
-      yield put(setNetworkConnectivity(isConnected(connectionInfo)))
+      const isNetworkConnected = isConnected(connectionInfo)
+      yield put(setNetworkConnectivity(isNetworkConnected))
+      Logger.setIsNetworkConnected(isNetworkConnected)
```

Comment:
In case changing the network state triggers an error, it's preferable to put it before the dispatch.
```suggestion
      const isNetworkConnected = isConnected(connectionInfo)
      Logger.setIsNetworkConnected(isNetworkConnected)
      yield put(setNetworkConnectivity(isNetworkConnected))
```

Reply:
yeah good idea, i swapped the order!